src/ngraph/runtime/cpu/op/bounded_relu.cpp

```diff
@@ -19,6 +19,16 @@
 using namespace std;
 using namespace ngraph;
 
+namespace ngraph
```

Comment:
Why add support to only two cpu ops and not all of them ?

Reply:
These two are utilizing the validator for `UnaryElementwiseArithmetic` operators and so required the update. I did not touch backend specific op validations in this PR unless they were using validators defined in ngraph core.